addons/website_slides/tests/test_slide_channel.py

```diff
@@ -274,6 +275,47 @@ def _get_completion_time_pdf(*args, **kwargs):
         self.assertEqual(13.37, round(slides_1.completion_time, 2))
         self.assertEqual(123.0, slides_2.completion_time)
 
+    def test_if_unlink_and_unpublish_channel_slides_sends_competition_email(self):
+        """When all content (slide.slide) are deleted from the course (slide.channel),
+        the course participants (slide.channel.partner) SHALL NOT receive completers email notification. """
```

Comment:
You may expect that the reader is familiar with the models here.
```suggestion
        """Check that course participants do not receive a "course completed" email when slides are deleted."""
```